unit-tests/live/d400/test-pipeline-set-device.py

```diff
@@ -0,0 +1,33 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2024 Intel Corporation. All Rights Reserved.
+
+# LibCI doesn't have D435i so //test:device D435I// is disabled for now
+# test:device D455
+
+import pyrealsense2 as rs
+from rspy import test
+
+gyro_sensitivity_value = 4.0
+
+with test.closure("pipeline - set device"):
+    cfg = rs.config()
```

Comment:
Used?

Reply:
Oh, now I see, why not declare before usage? it's C++ :)

Reply:
It's python :)
Same comment but that's OK for now.
In the future, with C++ and python it is better to declare the variable right  before usage.
This way if you remove the usage in the future, you will remember to remove the declaration too